tests/test_simulate.py

```diff
@@ -47,7 +47,7 @@ def model_setup(toy_model_exog_ltc):
             n_agents, dtype=np.int64
         ),  # all agents start as workers
         "married": np.zeros(n_agents, dtype=np.int64),
-        "ltc": np.zeros(n_agents, dtype=np.int64),
```

Comment:
In introduced an error with this change. Our code needs to be robust against dtype specifications. Same in the other test.

Reply:
We need to fix dtype in create exog mapping and also in the beginning of the simulation.